### PR TITLE
Assign FHIR and DICOM roles automatically

### LINKED ISSUE #3103: Assign Fhir Contributor and Dicom Data Owner roles to  Workspace Researcher automatically
## Description

As a TRE Workspace Owner
I want to assign everyone with Workspace Researcher role to Fhir Contributor/Dicom Data Owner role
So that I don't need to do it manually

### PR DESCRIPTION
**Pull request by LizaShak**
# Resolves #3103 

## What is being addressed

Assign everyone with Workspace Researcher role to Fhir Contributor/Dicom Data Owner role upon health services creation. 

## How is this addressed

- Get all entities assigned with Workspace Researcher role on the workspace client id
- Assign them with Fhir Contributor/Dicom Data Owner role  accordingly

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ COMPONENTS:
 | tre-service-guacamole-import-reviewvm | 0.2.0 |
 | tre-service-guacamole-linuxvm | 0.6.1 |
 | tre-service-guacamole-windowsvm | 0.6.0 |
-| tre-workspace-service-health | 4.1.9 |
+| tre-workspace-service-health | 0.1.0 |
```
Comment (tamirkamara) [inline CHANGELOG.md:71]:
```suggestion
| tre-workspace-service-health | 0.2.0 |
```

File: templates/workspace_services/health-services/porter.yaml [hunk 2]
```diff
@@ -68,9 +76,12 @@ outputs:
       - upgrade
 
 mixins:
-  - exec
   - terraform:
       clientVersion: 1.3.6
+  - az:
+      clientVersion: 2.37.0
+      extensions:
+        - azure-firewall
```
Comment (tamirkamara) [inline templates/workspace_services/health-services/porter.yaml:84]: Do you need the extension?

**Review by tamirkamara — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (LizaShak) [reply CHANGELOG.md:71]: It is the fix for the previous release, you said I should add it in this PR

**Pull request merged**